docs/source/api/plugin/usage-reporting.md

```diff
@@ -223,7 +217,9 @@ How often to send reports to Apollo. We'll also send reports when the report get
 </td>
 <td>
 
-We send a report when the report size will become bigger than this size in bytes (default: 4MB).  (This is a rough limit --- we ignore the size of the report header and some other top level bytes. We just add up the lengths of the serialized traces and signatures.)
```

Comment:
This is reference documentation, so I would prefer you keep the parenthesized caveat.